Gruntfile.js

```diff
@@ -633,6 +633,9 @@ module.exports = function(grunt) {
           'replace:update-app-constants',
           'generate-service-worker',
           'couch-compile:primary',
+          'browserify:admin',
```

Comment:
Is this making it so building webapp builds the admin app? Can you explain the thinking here?

Reply:
I'm making the `watch` task rebuild admin-app when webapp/js & shared libs files are updated because we use webapp/shared-libs code in the admin-app: https://github.com/medic/medic/blob/master/admin/src/js/main.js#L74

I was testing to see if search was working correctly in admin (we use the search service for all select2 contact search dropdowns, including the contact search when creating users in admin) and I got annoyed to have to build admin-app manually when I updated Search lib code.

Reply:
Isn't a better way to do this to add shared-lib into the list of watched files for `admin-js` task on line 593?

Reply:
Done!